src/proxy-ui-api/frontend/src/views/AddClient/ClientDetailsPage.vue

```diff
@@ -180,9 +185,9 @@ export default Vue.extend({
   data() {
     return {
       disableDone: false,
-      certificationService: undefined,
       filteredServiceList: [],
```

Comment:
This doesn't seem to be used anymore, maybe it should be removed?

Reply:
Removed it and also another unused value